src/App.js

```diff
@@ -1,24 +1,25 @@
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Home from "./components/Home"
+import Categories from "./components/Categories"
+import Nav from './components/Nav';
+import { Reviews } from './components/Reviews';
 
 function App() {
   return (
+    <BrowserRouter>
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+        <h1>NC Games</h1>
+        <Nav></Nav>
+      <nav>
+        <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/categories" element={<Categories />} />
+        <Route path="/reviews" element={<Reviews />} />
+        </Routes>
+      </nav>
```

Comment:
Semantic `<nav>` tags are great for use in the navbar, but shouldn't be used here to wrap a router